yt/frontends/ramses/hilbert.py

```diff
@@ -1,6 +1,7 @@
 from typing import Any, Optional
 
 import numpy as np
+import numpy.typing as nptype
```

Comment:
```suggestion
import numpy.typing as npt
```

you'll have to update the remainder of the file.